node-zerox/src/index.ts

```diff
@@ -12,6 +12,7 @@ import fs from "fs-extra";
 import os from "os";
 import path from "path";
 import pLimit, { Limit } from "p-limit";
+import { fileTypeFromFile } from "file-type";
```

Comment:
We might want to use another package like `mime-type`. Seems like this package is an ESM package. `This package is an ESM package. Your project needs to be ESM too.`

https://www.npmjs.com/package/mime-types